restcomm/restcomm.http/src/main/java/org/mobicents/servlet/restcomm/http/AccountsEndpoint.java

```diff
@@ -252,6 +283,35 @@ protected Response putAccount(final MultivaluedMap<String, String> data, final M
         }
     }
 
+    private Client createClientFrom(final Sid accountSid, final MultivaluedMap<String, String> data) {
+        final Client.Builder builder = Client.builder();
+        final Sid sid = Sid.generate(Sid.Type.CLIENT);
+        //TODO: need to encrypt this password because it's same with Account password.
+        // Don't implement now. Opened another issue for it.
+        //String password = new Md5Hash(data.getFirst("Password")).toString();
+        String password = data.getFirst("Password");
+
+        builder.setSid(sid);
+        builder.setApiVersion(getApiVersion(data));
+        builder.setFriendlyName(data.getFirst("FriendlyName"));
+        builder.setAccountSid(accountSid);
+        builder.setLogin(data.getFirst("Login"));
+        builder.setPassword(password);
+        builder.setStatus(Client.ENABLED);
+        builder.setVoiceUrl(getUrl("VoiceUrl", data));
```

Comment:
over here object data will never contain the above parameters. Better remove them.